### PR TITLE
fix(urls): Fix urls

### PR DESCRIPTION
**Pull request by Astrak**

### CODE REVIEW

File: src/lib/logOut.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { B2B_API_URL, BASE_URL } from "@/utils/constants";
+import { BASE_URL } from "@/utils/constants";
 
 export async function logOut(jwt: string) {
   document.cookie = "session-wallet=";
-  await fetch(`${B2B_API_URL}/users/logout`, {
+  await fetch(`${BASE_URL}/users/logout`, {
```
Comment (santiago-ab) [inline src/lib/logOut.ts:5]: this is an API request

**Review by santiago-ab — CHANGES_REQUESTED** [review]
Easy fix required

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3002",
```
Comment (majid-allianceblock) [inline package.json:6]: please revert this and just change it in your local

File: src/hooks/auth/getAuthorizationUrlSdk.ts [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
-    appId,
-    process.env.NEXT_PUBLIC_API_URL as string
-  );
+  const walletInfra = new WalletInfra(appId, B2B_API_URL);
   const url = walletInfra.generateAuthUrl(
-    `${process.env.NEXT_PUBLIC_BASE_URL}/home`,
+    `${BASE_URL}/home`,
     AuthProvider.GOOGLE
   );
+  console.log(url);
```
Comment (majid-allianceblock) [inline src/hooks/auth/getAuthorizationUrlSdk.ts:10]: remove it

**Review by majid-allianceblock — CHANGES_REQUESTED** [review]

**Pull request closed without merging**